lib/jsdom/living/helpers/style-rules.js

```diff
@@ -235,6 +237,29 @@ function getComputedValue(element, property) {
     const specifiedValue = getSpecifiedValue(element, property);
 
     // https://drafts.csswg.org/css-color-4/#resolving-other-colors
+    let value = "";
+    if (specifiedValue === "initial") {
+      value = initial;
+    } else if (specifiedValue === "inherit") {
+      if (element.parentElement) {
+        return getComputedValue(element.parentElement, property);
+      }
+      value = initial;
+    } else {
+      value = asciiLowercase(specifiedValue);
+    }
```

Comment:
Shouldn't we call `asciiLowercase()` even if `specifiedValue === "initial"` or `"inherit"`? What if we inherit a system color?

If I am right about this being a bug, please add a test that fails before and passes after.

Reply:
Applied asciiLowercase()